me.kozec.syncthingtk.yaml

```diff
@@ -41,27 +41,46 @@ modules:
       - /share/man
 
   - name: syncthing
+    disabled: false
     buildsystem: simple
+    build-options:
+        env:
+            GOBIN: /app/bin
+        #build-args:
```

Comment:
Ditto.

Reply:
nah. this is really needed for newer versions of syncthing. That is, when a new syncthing is released, it needs networks access. Instead of looking up how to make it work, I'd rather have the commented out version.